superset/alfred/utils.py

```diff
@@ -275,17 +298,23 @@ def retain_eml_to_alfred(ids, alfred_env, access_token, dates=None):
             catalog="hogwarts_pb",
         )
 
-        sql = f"""
-    SELECT id, urls, "from", to, message_id, reply_to, stream_id, subject, eml_path, bcc, cc, ministerial_authorization, classification, time
-    FROM  hogwarts_pb.harmonized.eml_metadata
-    WHERE id IN ('{ids_string}')
-    """
+        # excluding v6 columns due to utf encoding errors
```

Comment:
Nit: Can you raise a ticket to (eventually) fix this and include the ticket number here?

Reply:
I did actually fix the function to properly encode the v6, they remain left out at the moment because they are not useful still. Should I still create a ticket for readding them when the useful src and dst fields become available?

Reply:
Yesss, I think it'd be worthwhile.